### DISCUSSION TITLE
Do the username/keyHandle mappings files support comments of any kind?

### DISCUSSION THREAD

**Discussion by kurtlieber** [Yubico/pam-u2f#300] [Q&A]
I'd like to be able to note where each key comes from/what it's used for.  Are comments supported at all?

**Comment by LDVG** [comment]
Unfortunately, we have not defined any comment symbols for the authorization mapping file yet.

**Comment by thefoster** [comment]
> I'd like to be able to note where each key comes from/what it's used for. Are comments supported at all?

I was looking for the same as i'm using several Solo-/Yubukeys and found by simply trying that adding " #commentXY" at the end of each line in ~/.config/Yubico/u2f_keys just works fine, even on older Kubuntu 22.04
EDIT: was too fast on this and have to retract the hint.
It "worked" for about a n hour likely because of some caching and is not sustainable :-/
